package.json

```diff
@@ -21,6 +21,7 @@
     "deploy:development": "bash scripts/deploy_development.sh",
     "dist": "bash scripts/prepare_dist.sh",
     "clean": "rm -rf build",
+    "compile": "truffle compile",
```

Comment:
could just make the `truffle compile --all` and put it into the `test` method below too